packages/custom_lint/bin/custom_lint.dart

```diff
@@ -26,9 +27,53 @@ Future<void> entrypoint([List<String> args = const []]) async {
     return;
   }
 
+  final restArgs = [...result.rest];
+
+  final fileList = <String>[];
   final watchMode = result['watch'] as bool;
 
-  await customLint(workingDirectory: Directory.current, watchMode: watchMode);
+  // Coverting to absolute paths as the linter expects
+  for (var i = 0; i < restArgs.length; i++) {
+    if (path.isRelative(restArgs[i])) {
+      restArgs[i] = path.absolute(restArgs[i]);
+    }
+  }
+
+  // Populating fileList with all the files we can find
+  for (var i = 0; i < restArgs.length; i++) {
+    final pathStats = File(restArgs[i]).statSync();
+    if (pathStats.type == FileSystemEntityType.file) {
+      fileList.add(restArgs[i]);
+    } else {
+      final dir = Directory(restArgs[i]);
+      final fileEntities = dir.listSync(recursive: true);
+
+      for (final fileEntity in fileEntities) {
+        fileList.add(fileEntity.path);
+      }
+    }
+  }
+
+  // for (var i = 0; i < dirList.length; i++) {
```

Comment:
Can you remove this? :)

Reply:
Very sorry for this. Was so excited, just didn't look at that properly :)